client/src/Components/TodoForm.jsx

```diff
@@ -1,103 +1,79 @@
-import React, { useState, useEffect, useRef } from 'react'
-import axios from "axios";
+import React, { useState, useEffect, useRef } from 'react';
+import axios from 'axios';
 
 function TodoForm(props) {
-    const API_URL = process.env.REACT_APP_API_URL || "http://localhost:5000";
-    const [input, setInput] = useState('');
-    const [placeholderInput, setPlaceholderInput] = useState('');
+  const API_URL = process.env.REACT_APP_API_URL || 'http://localhost:5000';
+  const [input, setInput] = useState(props.edit ? props.edit.name : '');
+  const [titleInput, setTitleInput] = useState(props.edit ? props.edit.title : '');
 
-    const inputRef = useRef(null)
-    const placeholderInputRef = useRef(null);
+  const inputRef = useRef(null);
+  const titleInputRef = useRef(null);
 
-    const handlePlaceholderChange = e => {
-        setPlaceholderInput(e.target.value);
-    };
-    useEffect(() => {
-        inputRef.current.focus()
-    })
+  const handleChange = (e) => {
+    setInput(e.target.value);
+  };
 
-    const handleChange = e => {
-        setInput(e.target.value);
-    };
+  const handleTitleChange = (e) => {
+    setTitleInput(e.target.value);
+  };
 
-    const handleSubmit = (e) => {
-        console.log("im him")
-        e.preventDefault();
-        const token = localStorage.getItem('token');
-        console.log(token)
-        console.log("hello")
-        props.onSubmit({
-            id: Math.floor(Math.random() * 500000),
-            text: input,
-            placeholder: placeholderInput
-        });
-        // Create an object with the data you want to send to the server
-        const todoData = {   
-            name: input,
-            title: PlaceholderInput,
-        };
+  useEffect(() => {
+    inputRef.current.focus();
+  }, []);
 
-        // Send a POST request to the server
-        axios
```

Comment:
We don't need to use axios.post because we post to the server in the function `props.onSubmit`